CHANGELOG.md

```diff
@@ -10,8 +10,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 
-- Add new templates for `epic` issue and `feature request` - [4](https://github.com/IronFoundation/iOS-Woof/pull/4)
-- Add the possibility for sitters to update info about themselves - [52](https://github.com/IronFoundation/iOS-Woof/pull/52)
+- Add new templates for `epic` issue and `feature request` - [#4](https://github.com/IronFoundation/iOS-Woof/pull/4)
```

Comment:
nit:
I think it's not necessary to include a changelog entry for the changes to our workflow; should the entries be limited to the application only?

Reply:
We can discuss that, it depends on what we want to track just the progress of app changes or all repo changes.
But right now I just fixed the style, because we previously had the `#` sign before number of the PR.